### PR TITLE
Scripting object

### PR DESCRIPTION
**Pull request by r0x0r**

### CODE REVIEW

**Comment by shivaprsdv** [comment]
Review checklist: https://github.com/r0x0r/pywebview/issues/83#issuecomment-345848804.

@r0x0r To be sure it's working smooth, can we try fix the failing (macOS)
tests before merge?

**Comment by r0x0r** [comment]
@shivaprsdv I have fixed the configuration issue with OSX Python on Travis, but it appears the mouse helper function (that moves a mouse cursor a few pixels for Cocoa events to propagate) has ceased working. Let's look into this separately. I will merge this PR now.

**Comment by shivaprsdv** [comment]
Okay. I have added a remainder to #128. I have also fixed a small mistake in the
Travis config, on `master`.

File: webview/__init__.py [hunk 1]
```diff
@@ -253,8 +254,56 @@ def evaluate_js(script):
         raise Exception("Cannot call function: the webview has been closed")
 
 
+def set_js_api(api_instance):
```
Comment (shivaprsdv) [inline webview/__init__.py]: @r0x0r I only noticed this now, does this function work? IIRC this went obsolete somewhere
down the work on the branch.

**Pull request merged**